### ISSUE TITLE
Near queries don't trigger loaded hook

### ISSUE THREAD

**Issue by mastertinner** [strongloop/loopback-datasource-juggler#785]
When using the MongoDB connector, queries which use the "near" filter of the [GeoPoint class](https://docs.strongloop.com/display/LB/GeoPoint+class) (e.g. `{ "where": { "geo": { "near": "47,7" }}}`) don't trigger the `loaded`[operation hook](https://docs.strongloop.com/display/public/LB/Operation+hooks#Operationhooks-loaded). All other queries do trigger the hook.

This error doesn't occur when using the memory connector.

**Comment by jmyrland** [comment]
This is also the case for the mysql-connector.

The following code triggers the `loaded` operation hook:
```
Model.find({}, (error, result) => {  ...  });
```

The following code does not:
```
Model.find({ where: { something: 'something'} }, (error, result) => {  ...  });
```

Is there any way to trigger the `loaded` hook manually?

**Comment by Amir-61** [comment]
IIRC this is solved as part of [PR#1061](https://github.com/strongloop/loopback-datasource-juggler/pull/1061).
